esda/moran.py

```diff
@@ -509,9 +509,7 @@ def by_col(
 def Moran_BV_matrix(variables, w, permutations=0, varnames=None):
     """
     Bivariate Moran Matrix
-
```

Comment:
It seems that your editor removed blank lines in the docstring. Can you please put them back (all of them)?